README.md

```diff
@@ -75,64 +81,160 @@ which the standard toast aims to accomplish natively.
+- `theme`: one of `"default"`, ???, or ???, conveying the semantic priority of the toast, and influencing its styling (both default and user-provided)
+- `position`: one of `"top-left"`, `"top-center"`, `"top-right"`, `"middle-left"`, `"middle-center"`, `"middle-right"`, `"bottom-left"`, `"bottom-center"`, or `"bottom-right"`. The default is (if the attribute is omitted or set to an invalid value) is `"bottom-right"`.
+    - TODO: this seems stylistic; should it be controlled via CSS instead?
+    - TODO: these are physical positions; should we use [logical ones](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Basic_concepts) instead?
+- `closebutton`: a boolean attribute, determining whether an explicit close button is shown. By default toasts do not have a close button.
 
-These attributes and default values will be attached to the `<std-toast>` HTML element.
+All attributes will be reflected as properties on the element's JavaScript interface.
+For example:
 
-- [Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)
-- `open`: `Boolean` - default: false
-- `theme`: `String` - default: 'default' (??)
-- `postition`: `String` - default: 'bottom-right' (??)
+```js
+const toast = document.createElement('std-toast');
+console.log(toast.open); // false
+```
 
-*TODO(me): how to explain things that can be both attributes and properties (elt + js patterns)*
+#### Contents
 
-*TODO(me): where to put action and how to format it?*
+The `<std-toast>` element can be used as a generic container,
+but will work best (in terms of default styling and events) if the developer conforms to the following content model:
 
+- The first child node,
+  typically either a text node or a container like `<p>`,
+  provides a message.
+- If more than one child node is present,
+  then the last child element,
+  which can be either an `<a>` or a `<button>`,
+  provides a call to action.
 
-### Properties & Functions
+Thus, the following would all work well out of the box:
 
-The std-toast library will come with a `showToast` function,
-which will handle the case of opening a toast
-and optionally creating it as well in Javascript.
+```html
+<std-toast>Hello world!</std-toast>
 
-#### `showToast(message, options)`
+<std-toast><p>Hello world!</p></std-toast>
 
-Creates a toast object displaying the `message`
-with an object of the `options`.
+<std-toast>
+  <p>Hello world!</p>
+  <button>Click me!</button>
+</std-toast>
 
-The options the developers can configure are:
+<std-toast>
+  <p>Hello world!</p>
+  <a href="https://example.com/" target="_blank">Click me!</button>
+</std-toast>
+```
 
-- `duration`
-- `closeButton`
-- `multiple` (??)
-- `newestOnTop` (??)
+These can equivalently be created via JavaScript:
+
+```js
+// TODO: write this once we figure out the JS API for supplying HTML messages,
```

Comment:
My current thinking is using template strings and just passing those in, but I'm still not sure what issues that extra complexity would add.

Reply:
Yeah, my worry there is that APIs which take raw HTML are a bit of a security issue. But maybe it'd be fine to start there...